lib/deep-speech-interface.js

```diff
@@ -84,68 +84,80 @@ class DeepSpeechInterface {
       fs.mkdirSync(this._modelsDir);
     }
 
-    const localLMTxt = path.join(this._modelsDir, 'local_lm.txt');
-    const localLMArpa = path.join(this._modelsDir, 'local_lm.arpa');
-    const localLMBinary = path.join(this._modelsDir, 'local_lm.binary');
-    fs.writeFileSync(localLMTxt, finalGrammar.join('\n'));
+    const lmText = path.join(this._modelsDir, 'lm.txt');
+    fs.writeFileSync(lmText, finalGrammar.join('\n'));
 
-    const lmplz = spawnSync(
-      path.join(this._binDir, 'lmplz'),
+    const generateLm = spawnSync(
+      'python3',
       [
-        '--memory', '64M',
-        '--order', '2',
+        path.join(this._binDir, 'generate_lm.py'),
+        '--input_txt', lmText,
+        '--output_dir', this._modelsDir,
+        '--top_k', '500000',
```

Comment:
These values are different from before, or were previously not specified:
```
--top_k 500000          # previously unspecified
--arpa_order 5          # used to be 2
--max_arpa_memory 50%   # used to be 64M
--arpa_prune "0|0|1"    # previously unspecified

Reply:
This script is not useful in your case, you can directly generate manually your text file. This one is intented to run and ingest lots of data

Reply:
So I should just keep running `lmplz` and `build_binary` manually? Should I skip `filter`?

Reply:
You should just need those and not filter yes, there's no point in filtering here.

Reply:
Done.